apps/pi-drone-detector/scripts/download_datasets.py

```
@@ -0,0 +1,316 @@
+```
+(Class 1, centered, 10% width, 20% height of image)
+""")
+
+    print(f"  Created structure at {negatives_dir}")
+    print("  Add your coke can images to: images/")
+    print("  Add YOLO labels to: labels/")
+
+    return negatives_dir
+
+
+def create_combined_dataset(output_dir: Path, train_split: float = 0.8):
```

Ruff (0.14.10)

Unused function argument: `train_split`

(ARG001)